PwaUriHandler/explainer.md

```diff
@@ -110,28 +112,30 @@ These are the fields in each URI handler object:
 
 | Field     | Required / Optional | Description                                      | Default                           |
 |:----------|:--------------------|:-------------------------------------------------|:----------------------------------|
-| `base`    | Required            | Base portion of the URI to be handled. Can also be a complete URI. | N/A             |
-| `paths`   | Optional            | Array of allowed paths relative to `base`        | `[]`                              |
-| `excludePaths` | Optional       | Array of disallowed paths relative to `base`     | `[]`                              |
+| `host`    | Required            | host portion of the URI to be handled. Can also be a complete URI. | N/A             |
+| `paths`   | Optional            | Array of allowed paths relative to `host`        | `[]`                              |
+| `excludePaths` | Optional       | Array of disallowed paths relative to `host`     | `[]`                              |
+
+`host` must match valid hosts under [RFC 3986](https://tools.ietf.org/html/rfc3986). `path` and `excludePaths` must match valid paths. If a scheme is provided in `host`, it must be __https://__ and not __http://__. Otherwise, __https://__ will be assumed.
```

Comment:
what happens in the event the developer passes http://www.contoso.com?

Reply:
The handler object should not be registered. Edited.